### PR TITLE
Margin pairs

### PR DESCRIPTION
**Pull request by 5queezer**
Automatically update margin pairs on launch

### CODE REVIEW

File: src/trader.js [hunk 1]
```diff
@@ -945,7 +917,30 @@ async function UpdateOpenTrades() {
...
 }
 
+async function UpdateMarginPairs() {
+    return new Promise((resolve, reject) => {
+        axios
+            .get(
+                "https://www.binance.com/gateway-api/v1/friendly/margin/symbols"
+            )
+            .then((res) => {
+                let btconly = res.data.data.filter(
```
Comment (dargmuesli) [inline src/trader.js:927]: The BVA strategy only uses BTC quoted pairs, yes, but that isn't a requirement for strategies, is it @jsappme?
So I propose we should remove this filter.

File: src/trader.js [hunk 2]
```diff
@@ -945,7 +917,30 @@ async function UpdateOpenTrades() {
...
+                    (obj) => obj.isMarginTrade == true && obj.quote == "BTC"
+                )
+                let list = btconly.map((obj) => obj.symbol)
+                margin_pairs = list.sort()
+                console.log("Margin Pairs:", margin_pairs)
+                resolve(true)
```
Comment (dargmuesli) [inline src/trader.js:933]: Is there a reason `resolve` isn't called with the margin pairs array, but with just `true` instead, expect it being done like this in other parts of the code?
I'd propose to resolve with the actual data and reject with the error message below.

**Comment by dargmuesli** [comment]
@5queezer @jsappme any remarks on my comments?

File: src/trader.js (hunk 1 above)
Comment (jsappme) [reply src/trader.js:927]: yes feel free to remove this filter

**Pull request merged**